Wordle.Api/Wordle.Api/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+{

```

Comment:
Add DefaultConnection in ConnectionString so that if refences the localDb for local testing/implementation.

Reply:
Sorry, can you expand more on that, where specifically.